src/sentry/integrations/bitbucket_server/client.py

```diff
@@ -168,3 +183,27 @@ def get_auth(self):
             signature_method=SIGNATURE_RSA,
             signature_type="auth_header",
         )
+
+    def _get_values(self, uri, params):
+        values = []
+        done = False
+        start = 0
+
+        logger.info("Loading values for paginated uri %s", uri)
+
+        while not done:
+            logger.debug("Loading values for paginated uri %s starting from %s", uri, start)
+            new_params = params
```

Comment:
@marbon87 You should make a shallow copy here for each `new_params` so we don't mutate the original parameters.